README.md

```diff
@@ -110,10 +110,10 @@ this particular ICU library and `rust_icu` version combination.
 | ------------------ | -------- | -------- | --------- | ---------- | -------- |
 | 0.1                | ☟        | ☟        | ☟         | ☟          | ☟        |
 | 0.1.3              | 1        | 1        | 1; 2; 2+3 | 1          | 1        |
-| 0.1.4              | 1        | 1        | 1; 2; 2+3 | 1          | 1        |
+| 0.2.0              | 1        | 1        | 1; 2; 2+3 | 1          | 1        |
```

Comment:
Can you please add a line with `0.2` similar to the line `0.1`, and `0.2.0` below it?

Reply:
Done